### PR TITLE
Add badges with versions and statistics to README

### PR DESCRIPTION
**Pull request by terhorstd**
There is a huge number of badges available to highlight more aspects of the installation and use. This is can help to point people to the right versions and sources. I also added some social buttons, which we probably need to discuss.

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -1,10 +1,29 @@
...
+[![Conda downloads](https://img.shields.io/conda/dn/conda-forge/nest-simulator?logo=conda-forge&logoColor=white&color=blue)](https://anaconda.org/conda-forge/nest-simulator)
+[![Ubuntu version](https://img.shields.io/badge/ubuntu-v2.20.0%20(PPA)-blue?logo=debian)](https://nest-simulator.readthedocs.io/en/latest/installation/)
+[![Fedora package](https://img.shields.io/fedora/v/nest?logo=fedora)](https://src.fedoraproject.org/rpms/nest)
+[![Homebrew version](https://img.shields.io/homebrew/v/nest.svg?logo=apple)](https://formulae.brew.sh/formula/nest)
+
+[![Virtual applicance](https://img.shields.io/badge/VM-v2.20.0-blue?logo=CodeSandbox)](https://nest-simulator.readthedocs.io/en/latest/download.html#download-livemedia)
+[![Docker Image Version](https://img.shields.io/docker/v/nestsim/nest/2.20.0?label=docker&logo=docker)](https://hub.docker.com/r/nestsim/nest)
+[![Docker Image Size](https://img.shields.io/docker/image-size/nestsim/nest)](https://github.com/nest/nest-docker)
+[![Docker Pulls](https://img.shields.io/docker/pulls/nestsim/nest?label=pulls)](https://hub.docker.com/r/nestsim/nest)
```
Comment (jougs) [inline README.md:23]: I also doubt the 1.9k pulls of our Docker image. Please remove.

File: README.md [hunk 2]
```diff
@@ -1,10 +1,29 @@
...
+
+[![YouTube Video Views](https://img.shields.io/youtube/views/K7KXmIv6ROY?style=social)](https://www.youtube.com/results?search_query=nest-simulator+neurons)
+[![Twitter Follow](https://img.shields.io/twitter/follow/nestsimulator?style=social)](https://twitter.com/nestsimulator)
```
Comment (jougs) [inline README.md:26]: I'd like plain links without the counts better.

File: README.md [hunk 3]
```diff
@@ -1,10 +1,29 @@
-# The Neural Simulation Tool - NEST 
+# The Neural Simulation Tool - NEST
 
-[![Build Status](https://travis-ci.org/nest/nest-simulator.svg?branch=master)](https://travis-ci.org/nest/nest-simulator)
+[![Build status](https://img.shields.io/travis/nest/nest-simulator/master.svg?logo=travis-ci&logoColor=white)](https://travis-ci.org/nest/nest-simulator)
+[![Documentation](https://img.shields.io/readthedocs/nest-simulator?logo=readthedocs)](https://nest-simulator.org/documentation)
```
Comment (terhorstd) [inline README.md:4]:
```suggestion
[![Documentation](https://img.shields.io/readthedocs/nest-simulator?logo=readthedocs&label=Documentation)](https://nest-simulator.org/documentation)
```

File: README.md [hunk 4]
```diff
@@ -1,10 +1,29 @@
...
+[![CII Best Practices](https://bestpractices.coreinfrastructure.org/projects/2218/badge)](https://bestpractices.coreinfrastructure.org/projects/2218)
 [![License](http://img.shields.io/:license-GPLv2+-green.svg)](http://www.gnu.org/licenses/gpl-2.0.html)
 [![DOI](https://zenodo.org/badge/DOI/10.5281/zenodo.3605514.svg)](https://doi.org/10.5281/zenodo.3605514)
-[![Latest Version](https://img.shields.io/badge/latest%20version-2.20.0-brightgreen.svg)](https://github.com/nest/nest-simulator/releases/tag/v2.20.0)
-[![CII Best Practices](https://bestpractices.coreinfrastructure.org/projects/2218/badge)](https://bestpractices.coreinfrastructure.org/projects/2218)
+
+[![Latest release](https://img.shields.io/github/release/nest/nest-simulator.svg?color=brightgreen&label=latest%20release)](https://github.com/nest/nest-simulator/releases)
+[![GitHub contributors](https://img.shields.io/github/contributors/nest/nest-simulator?logo=github)](https://github.com/nest/nest-simulator)
+[![GitHub commit activity](https://img.shields.io/github/commit-activity/y/nest/nest-simulator?color=%23ff6633)](https://github.com/nest/nest-simulator)
```
Comment (terhorstd) [inline README.md:11]:
```suggestion
[![GitHub commit activity](https://img.shields.io/github/commit-activity/y/nest/nest-simulator?logo=github&color=%23ff6633)](https://github.com/nest/nest-simulator)
```

File: README.md [hunk 5]
```diff
@@ -1,10 +1,29 @@
...
+[![GitHub All Releases](https://img.shields.io/github/downloads/nest/nest-simulator/total?logo=github)](https://github.com/nest/nest-simulator/releases)
```
[the remark this answers is not in the document]
Comment (terhorstd) [reply README.md:12]:
```suggestion
```
agree, might not be too useful.

File: README.md (hunk 1 above)
Comment (terhorstd) [inline README.md:23]:
```suggestion
[![Ubuntu version](https://img.shields.io/badge/ubuntu-v2.20.0%20(PPA)-blue?logo=debian)](https://nest-simulator.readthedocs.io/en/latest/installation/)
[![Fedora package](https://img.shields.io/fedora/v/nest?logo=fedora)](https://src.fedoraproject.org/rpms/nest)
[![Conda version](https://img.shields.io/conda/vn/conda-forge/nest-simulator.svg?logo=conda-forge&logoColor=white)](https://anaconda.org/conda-forge/nest-simulator)
[![Homebrew version](https://img.shields.io/homebrew/v/nest.svg?logo=apple)](https://formulae.brew.sh/formula/nest)
[![Docker Image Version](https://img.shields.io/docker/v/nestsim/nest/2.20.0?label=docker&logo=docker&logoColor=white)](https://hub.docker.com/r/nestsim/nest)
[![Virtual applicance](https://img.shields.io/badge/VM-v2.20.0-blue?logo=CodeSandbox)](https://nest-simulator.readthedocs.io/en/latest/download.html#download-livemedia)
```

File: README.md [hunk 6]
```diff
@@ -1,10 +1,29 @@
-# The Neural Simulation Tool - NEST 
+# The Neural Simulation Tool - NEST
 
-[![Build Status](https://travis-ci.org/nest/nest-simulator.svg?branch=master)](https://travis-ci.org/nest/nest-simulator)
+[![Build status](https://img.shields.io/travis/nest/nest-simulator/master.svg?logo=travis-ci&logoColor=white)](https://travis-ci.org/nest/nest-simulator)
```
Comment (terhorstd) [inline README.md:3]:
```suggestion
```

File: README.md (hunk 2 above)
Comment (terhorstd) [reply README.md:20]: @steffengraber, should these social links better appear on the nest-simulator.org (only)?

**Review by jougs — APPROVED** [review]
Very nice!

File: README.md (hunk 2 above)
Comment (jougs) [reply README.md:20]: I would be fine with both solutions.

Comment (steffengraber) [reply README.md:20]: > @steffengraber, should these social links better appear on the nest-simulator.org (only)?

I would say yes.

**Review by steffengraber — APPROVED** [review]
For docker, conda, VM we have not only v2.20.0 but also other versions.
`community activity` is a number that means nothing to me at first. But all in all, very nice.

**Pull request merged**